next-desolates-map/components/SettingsPage/SettingsPage.tsx

```diff
@@ -0,0 +1,466 @@
+import { useContext, useState } from "react";
+import CameraMovement from "../../core/modules/CameraMovement";
+import { Clusters, SpaceRendererContext, Stages } from "../../pages/_app";
+import { searchForPlanetAndChangeStage } from "../RenderController/RenderController";
+import Image from "next/image"
+import pic from "../../public/profile.png"
+
+
+export default function SettingsPage() {
+    const {
+        changeStage,
+        p5,
+        cam,
+        setOngoingCamMov,
+        celestialObjects,
+        setCluster,
+        setSelectedPlanet,
+    } = useContext(SpaceRendererContext);
+
+    const [planetSearchInput, setPlanetSearchInput] = useState("");
+
+    function handleInputChange(e: any) {
+        setPlanetSearchInput(e.target.value);
+    }
+
+    function handleKeyDown(e: any) {
+        if (e.key === "Enter") {
+            console.log("ENTER PRESSED");
+            searchForPlanet();
+        }
+    }
+
+    function searchForPlanet() {
```

Comment:
This is a function copy pasted from another file, but not helpful here. Let's remove it :)

Reply:
Done.